packages/discovery-provider/ddl/migrations/0051_usdc_purchase_access.sql

```diff
@@ -0,0 +1,17 @@
+begin;
+  -- add columns to usdc_purchases
+  alter table usdc_purchases
+  add column if not exists for_stream_access boolean default true;
```

Comment:
I still don't understand why this column isn't the same enum as usdc_purchase_access_type?

Reply:
updated to use the enum everywhere